packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -163,6 +190,16 @@ const patternSource = computed(() => {
     &::selection {
       background: var(--selection-color);
     }
+
+    #{get-autofill-selectors("&")} {
+      background-color: transparent;
+      -webkit-text-fill-color: var(--onyx-color-text-icons-neutral-intense);
+
+      // many browsers use "!important" to set the autofill background so we need this
+      // transition workaround to make the background transparent
+      $one-year-in-seconds: 31536000s;
```

Comment:
There is actually an infinity constant in CSS :D 
- https://codersblock.com/blog/playing-with-infinity-in-css/
- https://developer.mozilla.org/en-US/docs/Web/CSS/calc#formal_syntax

Reply:
Nice!